site/content/technical-specifications.md

```diff
@@ -14,19 +14,10 @@ This page describes technical specifications for F5 NGINX Ingress Controller, su
 ## Supported NGINX Ingress Controller versions
 
 We recommend using the latest release of NGINX Ingress Controller. We provide software updates for the most recent release. We provide technical support for F5 customers who are using the most recent version of NGINX Ingress Controller, and any version released within two years of the current release.
-
-- Release 3.0.0 provides support for the `discovery.k8s.io/v1` API version of EndpointSlice, available from Kubernetes 1.21 onwards.
-- Release 2.4.2 is compatible with the Kubernetes Ingress v1 API, available in Kubernetes 1.19 and later.
-- Release 1.12 supports the Ingress v1beta1 API and continues to receive security fixes to support environments running Kubernetes versions older than 1.19. The v1beta1 Ingress API was deprecated in Kubernetes release 1.19, and removed in the Kubernetes 1.22.
-
----
-
-## Supported Kubernetes versions
-
 We test NGINX Ingress Controller on a range of Kubernetes platforms for each release, and list them in the [release notes]({{< relref "/releases.md" >}}). We provide technical support for NGINX Ingress Controller on any Kubernetes platform that is currently supported by its provider, and that passes the [Kubernetes conformance tests](https://www.cncf.io/certification/software-conformance/).
 
 {{< bootstrap-table "table table-bordered table-striped table-responsive" >}}
-| NIC Version | Supported Kubernetes Version | NIC Helm Chart Version | NIC Operator Version | NGINX / NGINX Plus version |
+| NIC Version | Kubernetes Version tested with | NIC Helm Chart Version | NIC Operator Version | NGINX / NGINX Plus version |
```

Comment:
```suggestion
| NIC version | Kubernetes versions tested  | NIC Helm Chart version | NIC Operator version | NGINX / NGINX Plus version |
```
"With" is implied, so doesn't need to be part of the title. Adjusted capitalisation of `version` to be consistent with sentence case, and made it plural in the second instances since a range is mentioned.